operations/workplace/groups.md

```diff
@@ -11,3 +11,21 @@ This page describes different leadership groups at Mattermost.
 
 * Consists of MLT plus heads of key functions selected by MLT members to represent team
 * Included in [MLT Extended](https://community.mattermost.com/private-core/channels/mlt--extended) channel
+
+## Leadership Roles 
+
+Members of any management group are expected to embody and model leadership values and our [remote principles](https://handbook.mattermost.com/operations/workplace/people/working-at-mattermost#we-run-as-a-distributed-high-performance-team-using-a-few-principles). 
+
+### Director group 
+
+**What's new at Director level?**
+* **Owns vital functions of the business** - Engineering (headcount, scope, leadership, typically manager of managers), marketing (budget, strategy, scale of execution), sales (territory, material quotas, headcount), operations (vital systems, expertise, liability, budget, headcount) 
+* **Strong ability to manage one or more functions with minimal direction** - Trusted to create their own context aligned to company priorities, circle back on critical questions, develop goals, 3-6 month plan, anticipates obstacles, cadence to deliver results quickly and effectively 
+* **Cultural role model** - Recognized as a leader who embodies mission, six leadership principles (know them by heart), and “customers, company, team, self” mindset. Everyone would agree that if we could clone this person 100 times we would be an amazing company. 
+* **Ability to deliver cross-organizational solutions** - Owns problems to be solved through cross-organizational efforts and creates teams-of-teams to co-deliver solutions based on clearly defined vision, plans, cadences and outputs. Influences execs with data and logic. 
+* **Exceptionally strong recruiter, manager, coach upleveling their team and other teams** - Can source, hire, manage and continually uplevel teams and functions. Sought as a resource/mentor cross-organizationally to help in recruiting, vetting, coaching, managing. 
+ * **Source, hires, develops, manages, up-levels individuals and teams** - Experienced manager to hire and develop the best. Continually improves team and individual performance month-by-month. Recognized cross-organizationally as a strong leader and coach that people seek for advice. 
+* **Self-aware and builds teams as sum of strengths** - Strong sense of strengths and opportunities for self and team, and organizes and recruits to create organizations that are the sum of their strengths. Vocally self-aware. 
+* **Owns vital functions of the business** - Engineering (headcount, scope, leadership, typically manager of managers), marketing (budget, strategy, scale of execution), sales (territory, material quotas, headcount), operations (vital systems, expertise, liability, budget, headcount) 
```

Comment:
@natalie-hub - Line 22 and Line 29 are repeats. Not sure I agree with the way the information is laid out.

Reply:
Only comment is the repeated content in 22 and 29.